adapters/http-vertx-quarkus/pom.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.hono</groupId>
+    <artifactId>hono-adapters</artifactId>
+    <version>1.4.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>hono-adapter-http-vertx-quarkus</artifactId>
+  <name>Hono HTTP Adapter</name>
+
+  <description>A protocol adapter exposing Hono's Telemetry &amp; Event APIs via HTTP.</description>
+  <url>https://www.eclipse.org/hono</url>
+
+  <properties>
+    <quarkus.container-image.group>eclipse</quarkus.container-image.group>
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+    <surefire-plugin.version>2.22.1</surefire-plugin.version>
```

Comment:
FMPOV we do not need this

Reply:
I thought I already removed it. Thanks for spotting it.